env.d.ts

```diff
@@ -0,0 +1,8 @@
+declare module '@env' {
```

Comment:
is this supposed to be pushed? im not sure if we're supposed to be able to see env files (will ask kabir)

Reply:
I added a gitignore! Sorry!

Reply:
hmmm it was already in gitignore. please lmk if this happens again!